src/ecc/curves.iced

```diff
@@ -208,6 +219,18 @@ exports.Curve25519 = class Curve25519 extends Curve
 
   #----------------------------------
 
+  # Serialize single curve coordinate (usually private key X) to MPI
+  # buffer, with 2-byte header encoding bit-size of the coordinate.
+  coord_to_mpi_buffer : (p) ->
+    byte_size = @mpi_coord_byte_size()
+    throw new Error "Invalid p.length: got #{p.length} expected #{byte_size}." if p.length != byte_size
```

Comment:
This would only happen if someone gave some invalid `X` to `Priv` in ECDH, we shouldn't generate invalid keys or even load them (it would error out during loading). I just preferred having an error there, because if there is a bug somewhere else you end up with key that serializes but fails with some "invalid checksum" in other programs and that is hard to figure out.